vc/vc.go

```diff
@@ -0,0 +1,198 @@
+package vc
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/tbd54566975/web5-go/dids/did"
+	"github.com/tbd54566975/web5-go/jwt"
+)
+
+// these constants are defined in the W3C Verifiable Credential Data Model specification for:
+//   - [Context]
+//   - [Type]
+//
+// [Context]: https://www.w3.org/TR/vc-data-model/#contexts
+// [Type]: https://www.w3.org/TR/vc-data-model/#dfn-type
+const (
+	BaseContext = "https://www.w3.org/2018/credentials/v1"
+	BaseType    = "VerifiableCredential"
+)
+
+// DataModel represents the W3C Verifiable Credential Data Model defined [here]
+//
+// [here]: https://www.w3.org/TR/vc-data-model/
```

Comment:
```suggestion
// DataModel represents the W3C Verifiable Credential Data Model
//
// [here]: https://www.w3.org/TR/vc-data-model/
```

or whatevs, but you appear to imply there should be a link in the `[here]` but there isn't

Reply:
oh nevermind, I see your pattern here, disregard!